commcare_connect/opportunity/tests/test_tables.py

```
@@ -4,15 +4,78 @@
+def test_invoice_deliveries_table_total_folds_in_org_pay():
+    completed_work = CompletedWorkFactory(
+        saved_payment_accrued=40,
+        saved_org_payment_accrued=10,
+        saved_payment_accrued_usd=4,
+        saved_org_payment_accrued_usd=1,
+    )
+    table = InvoiceDeliveriesTable("KES", [completed_work], show_org=True)
+
+    rows = list(table.as_values())
+    headers, values = rows[0], rows[1]
+    row = dict(zip(headers, values))
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)